auth-web/src/store/modules/org.ts

```diff
@@ -5,7 +5,7 @@ import { AccountSettings } from '@/models/account-settings'
 import { EmptyResponse } from '@/models/global'
 import InvitationService from '@/services/invitation.services'
 import OrgService from '@/services/org.services'
-import { UserInfo } from '@/models/userInfo'
+import { UserInfo } from 'sbc-common-components/src/models/userInfo'
```

Comment:
While we move this file to common, can we also rename to something other than `userInfo`? Maybe `kcUserProfile`?

Reply:
If this too big of a change, no problem - it's not high priority.

Reply:
I agree, that name makes sense.
will change that

Reply:
done